lib/omnibus/software.rb

```diff
@@ -653,7 +654,7 @@ def with_standard_compiler_flags(env = {}, opts = {})
           arch_flag = windows_arch_i386? ? "-m32" : "-m64"
           opt_flag = windows_arch_i386? ? "-march=i686" : "-march=x86-64"
           {
-            "LDFLAGS" => "-L#{install_dir}/embedded/lib #{arch_flag} -fno-lto",
+            "LDFLAGS" => "-L#{install_dir}/embedded/lib #{arch_flag} -Wl,-rpath,#{install_dir}/embedded/lib",
```

Comment:
CRIT

Unsure if this helps...  it probably doesn't but this is a codechange that did get me past that build issues.  Don't know if it just changed timing of race conditions....